internal/kokoro/vet.sh

```diff
@@ -26,7 +26,8 @@ git diff go.sum | tee /dev/stderr | (! read)
 pwd
 
 # Look at all .go files (ignoring .pb.go files) and make sure they have a Copyright. Fail if any don't.
-git ls-files "*[^.pb].go" | xargs grep -L "\(Copyright [0-9]\{4,\}\)" 2>&1 | tee /dev/stderr | (! read)
+find . -type f -name "*.go" ! -name "*.pb.go" -exec grep -L "\(Copyright [0-9]\{4,\}\)" {} \; 2>&1 | tee /dev/stderr | (! read)
```

Comment:
Might want to add `*.sh` to this considering that was the file type we missed in a previous CL?

Reply:
Good call. Will do separately.